### PR TITLE
model installer -- Prevent crashes on malformed models

### PR DESCRIPTION
**Pull request by lstein**
This small patch improves the stability of `invokeai-*` scripts by avoiding crashes in the model manager while scanning the models directory for new and removed models.

### CODE REVIEW

File: invokeai/frontend/web/package.json
Comment (psychedelicious) [inline invokeai/frontend/web/package.json:1]: Same comment as on #3608 regarding what I suspect are accidental changes to `package.json`

**Review by psychedelicious — CHANGES_REQUESTED** [review]

File: invokeai/frontend/web/package.json
Comment (lstein) [reply invokeai/frontend/web/package.json:1]: Rolled back.

**Comment by lstein** [comment]
@psychedelicious I've made the requested change to rollback packages.json.

**Comment by psychedelicious** [comment]
@lstein sorry but `yarn.lock` is still in this PR, can you please remove it

**Review by psychedelicious — APPROVED** [review]
I'm approving but please remove `yarn.lock` before merging

**Pull request merged**